packages/components/src/SidePanel/SidePanel.component.js

```diff
@@ -132,7 +140,7 @@ function SidePanel({
 							id: getActionId(id, action),
 							bsStyle: 'link',
 							role: 'link',
-							className: classNames(theme.link, action.className),
+							className: action.className,
```

Comment:
Since you removed `theme.link`, you don't need that anymore.
`action.className` is already in the resulting object with the `Object.assign({}, action)`
You can remove this line